recipes/assimp/5.x/conanfile.py

```diff
@@ -141,14 +160,12 @@ def _depends_on_zlib(self):
 
     def requirements(self):
         # TODO: unvendor others libs:
-        # - clipper (required by IFC importer): can't be unvendored because CCI
-        #   has 6.4.2, not API compatible with 4.8.8 vendored in assimp
         # - Open3DGC
-        # - openddlparser
         if tools.Version(self.version) < "5.1.0":
             self.requires("irrxml/1.2")
         else:
             self.requires("pugixml/1.11")
+            self.requires("openddl-parser/cci.20211217")
```

Comment:
required by opengex importer only (`with_opengex=True`)

Reply:
FYI, I've opened an issue long time ago to implement fine-grained dependencies handling in Assimp (and easier external dependencies usage): https://github.com/assimp/assimp/issues/3213